packages/contracts-bedrock/deploy/016-SetAddressManager.ts

```diff
@@ -0,0 +1,33 @@
+import assert from 'assert'
+
+import { ethers } from 'ethers'
+import { DeployFunction } from 'hardhat-deploy/dist/types'
+import { awaitCondition } from '@eth-optimism/core-utils'
+import '@eth-optimism/hardhat-deploy-config'
+import 'hardhat-deploy'
+
+import {
+  getDeploymentAddress,
+  doOwnershipTransfer,
+} from '../src/deploy-utils'
+
```

Comment:
Can we do this in the post hook of the proxy admin?

Reply:
Yes, just had to bump the AddressManager (to step 0) to be deployed before the ProxyAdmin (now step 1).